systemverilog-plugin/tests/debug-flag/debug-flag.tcl

```diff
@@ -0,0 +1,16 @@
+yosys -import
+if { [info procs read_uhdm] == {} } { plugin -i systemverilog }
+yosys -import  ;# ingest plugin commands
+
+set TMP_DIR /tmp
+if { [info exists ::env(TMPDIR) ] } {
+  set TMP_DIR $::env(TMPDIR)
+}
+
+# Testing simple round-trip
+read_systemverilog -debug -odir $TMP_DIR/debug-flag-test -defer $::env(DESIGN_TOP)-pkg.sv
```

Comment:
`-defer` is here because I did copy another test that did already work. I just added `-debug`/`-report`. I don't think those two flags are affected that much by defer/link or their absence.

Reply:
I don't think so either, but this does make the test case larger (and potentially harder to debug) by splitting the test over multiple files.